src/components/NavBar/index.js

```diff
@@ -83,42 +103,90 @@ const QAButton = ({ intl, changeDropDownDirection }) => (
 const NavBarRightButtonContainer = styled.div`
   height: 70px;
   line-height: 70px;
-  position: absolute;
-  right: 70px;
-  top: 0px;
+  text-align: center;
+  color: white;
   padding-left: 20px;
   padding-right: 20px;
   border-left: 1px solid rgba(0,0,0,0.2);
+  display: flex;
 `;
 const NavBarRightButton = styled.div`
   height: 30px;
   margin: 20px auto;
   line-height: 30px;
-  text-align: center;
-  color: white;
+  display: flex;
 `;
+const SearchBarFont = styled.div`
+  color: rgba(255, 255, 255, 0.65);
+  padding-left: 14px;
+`;
+const DivSearchBarField = styled.div`
+  margin: auto;
+  display: flex;
+  width: 60%;
+`;
+
+const SearchBarField = ({ intl, classes, store: { search }, onClose, onSearchBarKeyDown }) => (
```

Comment:
same, dont pass store onClick listeners. just access the store func directly from the component.